pkg/fanal/analyzer/language/java/gradle/lockfile.go

```diff
@@ -0,0 +1,48 @@
+package gradle
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/gradle/lockfile"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+
+	"golang.org/x/xerrors"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&gradleLockAnalyzer{})
+}
+
+const (
+	version        = 1
+	fileNameSuffix = "gradle.lockfile"
```

Comment:
I can see on GitHub, that user have lock files with another names. is it ok?

Reply:
It is suffix of filename.
If I understand correctly - we need to check only this suffix.
https://github.com/aquasecurity/trivy/blob/6dda1d069bc71e07225157c319f9fbfa9e7ec588/pkg/fanal/analyzer/language/java/gradle/lockfile.go#L39